README.md

```diff
@@ -0,0 +1,19 @@
+# Picture in Picture Chrome Extension
+
+This extension demonstrates the Picture in Picture API in Chrome.
+
+**Note:** This only works in Chrome 68 or later.
+
+**To Install and Use It
+1. In `Chrome://flags` enable the following flags:
+   * \#enable-experimental-web-platform-features
+   * \#enable-surfaces-for-videos
+   * \#enable-picture-in-picture
+
+1. Download and extract the [extension zip file](https://github.com/beaufortfrancois/picture-in-picture-chrome-extension/archive/master.zip).
+1. In `Chrome://extensions` toggle **Developer mode** (upper-right corner) if it is not already on.
+1. To load the extension, click **Load Unpacked**.
+1. In the dialog box that appears locate and select the `src/` folder from the directory where you unpacked the zip file.
+1. Navigate to any YouTube video such as this one of [a band in Prague](https://www.youtube.com/watch?v=tbcnzyDQE1U).
```

Comment:
Mention that you have to click the extension icon to trigger PiP?
(It's fairly logical, but not self-explanatory.)